### PR TITLE
simpler return values from assert*Response

### PR DESCRIPTION
**Pull request by ttung**
Thanks to @mackey22 for the suggestion.

### CODE REVIEW

File: tests/test_api.py [hunk 1]
```diff
@@ -56,21 +56,19 @@ def upload_file(self, bundle_file):
                 source_url=bundle_file.url
             )
         )
-        response_data = json.loads(response[1])
-        self.assertIs(type(response_data), dict)
-        self.assertIn('version', response_data)
-        return response_data['version']
+        self.assertIs(type(response_json), dict)
```
Comment (Mackey22) [inline tests/test_api.py]: Looks like you wanted the middle return value here. Are they the same thing?

File: tests/test_api.py [hunk 2]
```diff
@@ -91,12 +89,11 @@ def put_bundle_payload(bundle):
         return payload
 
     def get_bundle_and_check_files(self, bundle):
-        response = self.assertGetResponse(
+        _, _, response_json = self.assertGetResponse(
```
Comment (Mackey22) [inline tests/test_api.py]: Same as above

File: tests/test_api.py (hunk 1 above)
Comment (ttung) [inline tests/test_api.py]: The text of the response is response[1].
The text of the response parsed into python structures from JSON is response[2].

I removed the call to `json.loads()`, so I just use response[2] directly.

File: tests/test_api.py (hunk 2 above)
Comment (ttung) [inline tests/test_api.py]: ditto.

**Comment by ttung** [comment]
Changed to a typed NamedTuple.  Ready for review.

**Pull request merged**